src/styles/index.ts

```diff
@@ -4025,6 +4025,17 @@ const styles = (theme: ThemeColors) =>
             verticalAlign: 'middle',
         },
 
+        chatBubbleCell: {
+            width: 24,
+            height: 20,
+            verticalAlign: 'middle',
+        },
+
+        chatBubbleCellText: {
+            fontSize: 9,
```

Comment:
Can you see if we have an existing font size variable for this? We call it `fine print` in Figma.

Reply:
It should also be bold I believe...